src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      state.books.push(
+        {
+          ...payload,
+          author: 'Suzanne Collins',
+          progress: 0,
+          state: 'Chapter 1',
+        },
+      );
+    },
+    removeBook: (state, { payload }) => {
+      /* eslint-disable no-param-reassign */
```

Comment:
- Good job done so far, but it is not advisable to disable linters, remember that linters help us ensure that we write code that is easy to maintain and meets global standards, please fix this 🙏

Reply:
@mckent05, you are correct about linters but as I have mentioned in the PR description this rule throws error when I mutate the state object which is totally allowed and is a main feature of redux toolkit.